examples/neo4j_integration/01_rdflib_neo4j_parse_grouped.py

```diff
@@ -0,0 +1,36 @@
+from rdflib import Graph
+from rdflib_neo4j import Neo4jStoreConfig, Neo4jStore, HANDLE_VOCAB_URI_STRATEGY
+
+example_file = "foaf.jelly"
+
+# Please introduce your credentials
+aura_db_uri = "your_db_uri"
+aura_db_username = "neo4j"
+aura_db_pwd = "your_db_pwd"
+
+# Prepare the authentication data to the Aura database
+auth_data = {
+    "uri": aura_db_uri,
+    "database": "neo4j",
+    "user": aura_db_username,
+    "pwd": aura_db_pwd,
+}
+
+# Prepare the configuration for neo4j store object
+config = Neo4jStoreConfig(
+    auth_data=auth_data,
+    handle_vocab_uri_strategy=HANDLE_VOCAB_URI_STRATEGY.IGNORE,
+    batching=True,
+)
+
+# Make a graph with Neo4jStore object
+Graph_data = Graph(store=Neo4jStore(config=config))
+
+# Your reference to data in rdf compatible format in batches
+data_grouped_example = [example_file for _ in range(10)]
+
+for idx, data in enumerate(data_grouped_example):
+    Graph_data.parse(data, format="jelly")
+    print(f"File with index {idx} has been successfuly parsed.")
+Graph_data.close(True)
+print("All done.")
```

Comment:
```suggestion
Graph_data = Graph(store=Neo4jStore(config=config))
Graph_data.parse(example_file, format="jelly")
Graph_data.close(True)
```

Also, could we use, let's say metadata.jelly from RiverBench? so that the example is self-contained

Reply:
also please rename the `Graph_data` var...

Reply:
Can you show at the end a small Cypher query to demonstrate it was loaded?

Reply:
Ok, now we use the metadata.jelly, above code implied + we do a check with print whether the data is actually in our database.